pkg/customresource/actionset.yaml

```diff
@@ -11,49 +11,78 @@ spec:
     singular: actionset
   scope: Namespaced
   versions:
-  - name: v1alpha1
-    served: true
-    storage: true
-    schema:
-      openAPIV3Schema:
-        properties:
-          apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
+    - name: v1alpha1
+      served: true
+      storage: true
```

Comment:
We generate this yaml right?
Can we just update the new entry manually here?
We can generate the yaml but just copy the snipped for `RepositoryServer` and copy that here. We wont see all these unnecessary changes here.
Or if we are sure that this wouldn't result into any problem, which it wont I think, we can go ahead with this as well.

Reply:
Sure will try that !